CHANGELOG/CHANGELOG-2.9.md

```diff
@@ -0,0 +1,242 @@
+# Release notes for v2.9.0
+
+[Documentation](https://kubernetes-csi.github.io)
+
+# Changelog since v2.8.0
+
+## Urgent Upgrade Notes 
+
+### (No, really, you MUST read this before you upgrade)
+
+- Action required: deprecated `kubelet-registration-probe` and removed related code. node-driver-registrar on Windows now requires Kubernetes with [fixed handling of registration sockets](https://github.com/kubernetes/kubernetes/pull/110075) that is part of Kubernetes v1.23.10, v1.24.4, v1.25.0, and newer. There are no changes on Linux, i.e. Kubernetes v1.13 is still the minimal one. (#316, @mattcary)
```

Comment:
Can you also update the table here to make the recommended version v1.23.10?  https://github.com/kubernetes-csi/node-driver-registrar#compatibility

Reply:
I added a new column for the recommended version. And fixed links to doc.